package.json

```diff
@@ -79,7 +79,8 @@
     "eslint-config-prettier": "^4.1.0",
     "eslint-plugin-import": "^2.16.0",
     "lint-staged": "^8.1.5",
-    "prettier": "^1.16.4"
+    "prettier": "^1.16.4",
+    "sinon": "^7.3.0"
```

Comment:
Do you think we could use this? https://github.com/jamiebuilds/ninos

Only to be consistent with the chosen test engine. It seems they are taken it seriously: https://github.com/avajs/ava/issues/1825

Reply:
Done